Formula/dotnet.rb

```diff
@@ -24,7 +24,22 @@ class Dotnet < Formula
   depends_on "icu4c"
   depends_on "openssl@1.1"
 
+  uses_from_macos "krb5"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "llvm" => [:build, :test]
+    depends_on "libunwind"
```

Comment:
`llvm` also provides a `libunwind`. Since we already have an `llvm` dependency, maybe we can use the one that comes with `llvm`?

Reply:
Not sure if `llvm` is a runtime dependency though.
Right now, there is no linkage to `llvm`, but there is linkage to `libunwind`.

Used Arch as reference https://archlinux.org/packages/community/x86_64/dotnet-runtime/

Reply:
Ah yes, that's fine then. `libunwind` is a lighter runtime depdendency anyway.

Reply:
The `libunwind` library in LLVM doesn't provide all the features of the standalone `libunwind`.  I remember getting very confused when trying to work on `dotnet` before, as I assumed that the LLVM `libunwind` would provide that functionality but I think it was failing.  Ultimately that was why a contributor ended up having to add the standalone `libunwind` as a formula.